README.MD

```diff
@@ -22,12 +21,6 @@ To build the client, run `make`.
 
 You can see usage of the generated code in the `examples` directory. In order to try, export `METAL_AUTH_TOKEN` token and execute the code, e.g.
 
-```
```

Comment:
This block can be added back in, with `plans.go`.

I can also look into adding the projects example back in. It is convenient that examples are included in the generated docs (even if they need module path changes)